ui_tests/src/test/resources/com/saucedemo/account/signInPositive.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Sign In with valid data
+
+  @signIn
+  Scenario: Sign In with valid user
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+
+
+  @signIn
+  Scenario: Sign In with valid password
+    Given I open start page
+    When I set 'performance_glitch_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then I am on Products page
```

Comment:
@masrur18 This test would fail with the following error:
![image](https://user-images.githubusercontent.com/3727228/77768352-bff12780-7018-11ea-9988-5324adb892cf.png)
There are two step definitions that allow you to click on Sign In button:
`I click 'Sign In' button on Sign in page expecting failure` : it's expecting you to remain on the Sign In page and not being logged in. It fails if you are getting redirected to Products page (i.e. you've signed in)
and `I click 'Sign In' button on Sign in page` : it's expecting you to get redirected to Products page (i.e. get signed in)
Please make sure you are using the right one in this scenario.